src/style.css

```diff
@@ -1,3 +1,109 @@
-body {
+/* General reset */
+* {
   margin: 0;
 }
+
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.menu {
+  display: flex;
+  padding-left: 0;
+}
+
+.menu__item {
+  list-style: none;
+  align-items: center;
+  display: flex;
+}
+
+.menu__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.menu__link {
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
+  position: relative;
+  font-weight: 500;
+}
+
+ .is-active,
+.menu__link:hover {
+  color: var(--color-blue);
+}
```

Comment:
use prettier to format your code properly, it seems something is wrong with spacing here